common/utils.py

```diff
@@ -63,8 +63,11 @@ def external_service_request(endpoint, data_json=None, session=None, method='POS
                 LOGGER.debug("JSON: %s", str(data_json))
                 # Do not retry for 4xx HTTP codes
                 if 400 <= response.status_code < 500:
-                    if service_access is not None and response.status_code == 404:
-                        service_access.status = True
+                    if service_access is not None:
+                        if response.status_code == 404:
+                            service_access.status = True
+                        elif response.status_code in [401, 403]:
```

Comment:
Why not use something like

```
if response.status_code == 404:
    service_access.status = True
else:
    service_access.status = False
```

to set a default state even for more states?

Reply:
sure 😄